### ISSUE TITLE
Undefined index: prevMinWidth

### ISSUE THREAD

**Issue by piotrpog** [wbrowar/craft-grid#1]
Just started testing out plugin, entered some random data to field setting and got error. Unfortunately im not really sure which thing caused this error specifically.
```
Undefined index: prevMinWidth

81828384858687888990919293949596979899                         * @inheritdoc
     */
    public function afterSave(bool $isNew)
    {
        if (!$isNew) {
            // Check for updated breakpoints
            $layout = Json::decodeIfJson($this->layout);
            $newMinWidths = [];
            foreach ($layout['breakpoints'] as $breakpoint) {
                if ($breakpoint['prevMinWidth'] !== $breakpoint['minWidth']) {
                    $newMinWidths[] = [
                        'old' => $breakpoint['prevMinWidth'],
                        'new' => $breakpoint['minWidth'],
                    ];
                }
            }
            // Update all elements that have field
            if (count($newMinWidths) > 0) {
                // Get layouts that include this field
```

**Comment by wbrowar** [comment]
Ah, okay. I know what's going on there. I'll fix that and get an update out today.

Thanks!

**Comment by wbrowar** [comment]
This is fixed in [465b24b](465b24bc5bfb3cbea640ab8fd3a28f3528bcbf69)
